google/cloud/pubsub/samples/samples.cc

```diff
@@ -2135,9 +2136,11 @@ void AutoRun(std::vector<std::string> const& argv) {
   GetTopic(topic_admin_client, {project_id, topic_id});
 
   std::cout << "\nRunning UpdateTopic() sample" << std::endl;
-  ignore_emulator_failures([&] {
-    UpdateTopic(topic_admin_client, {project_id, topic_id});
-  });
+  ignore_emulator_failures(
+      [&] {
+        UpdateTopic(topic_admin_client, {project_id, topic_id});
+      },
+      StatusCode::kInvalidArgument);
```

Comment:
So the new emulator implementation has a bug dealing with the `schema_settings` field mask? It is not our code that is at fault?

Reply:
yes, b/135688497 It used for fail with `kUnimplemented`, now it fails with `kInvalidArgument` when the field is "labels".  🤷